index.js

```diff
@@ -1,14 +1,21 @@
 const compiler = require('vue-template-compiler');
 const transpile = require('vue-template-es2015-compiler');
+const parse5 = require('parse5');
+const consolidate = require('consolidate');
```

Comment:
This isn't being used in index.js, could you take it out? It's not a huge library but taking it out speeds up the measly 22 tests by 4 tenths.